bin/browser.js

```diff
@@ -7,7 +7,10 @@ const callChrome = async () => {
     let page;
 
     try {
-        browser = await puppeteer.launch({ args: request.options.args || [] });
+        browser = await puppeteer.launch({
+            ignoreHTTPSErrors: request.options.ignoreHttpsErrors,
```

Comment:
To be honest, I wasn't sure if I should use a generic options array that gets merged into the config options array, or if it's okay to add the single option to the one line. There aren't many additional options (beyond the args) so I thought this was more readable. I'm interested if you have a strong opinion either way.